package.json

```diff
@@ -4,20 +4,39 @@
   "license": "Apache-2.0",
   "private": true,
   "engines": {
-    "node": ">=12"
+    "node": ">=20.0.0"
   },
   "workspaces": [
     "pkg/*"
   ],
   "dependencies": {
     "@rancher/components": "0.2.1-alpha.0",
-    "@rancher/shell": "^2.0.1",
-    "@types/lodash": "4.14.184",
-    "core-js": "3.21.1"
+    "@rancher/shell": "^3.0.0-rc.8",
+    "cache-loader": "^4.1.0",
+    "node-polyfill-webpack-plugin": "^3.0.0"
+  },
+  "devDependencies": {
+    "color": "4.2.3",
+    "ip": "2.0.1",
+    "@types/semver": "^7.5.8",
+    "vue3-resize": "0.2.0"
```

Comment:
none of the listed dev deps are used in the source code

Reply:
I'll reply to this comment but it should cover all the other ones you had about dependencies.

All of these are used in dashboard, but aren't dependencies of the `shell` library. I've promoted a discussion with the UI devs that worked on the Vue3 migration and we are evaluating moving them to `shell` and not be needing to add them at extension level, since they are `shell` dependencies at the end of the day.

As for the `types/lodash`, it's already a dependency of `shell` hence why we don't need to include it on Stackstate.

Let me consolidate all of this and we will see what we are left with 🙏